package.json

```diff
@@ -1,7 +1,7 @@
 {
-    "name": "MapStoreExtension",
+    "name": "longitutdinal_profile",
```

Comment:
Can you modify it to something like `LongitudinalProfile` ?

Reply:
Sure, but it complains then that it doesn't match regex. That's the only reason I didn't use camel case 😄

Reply:
Some weird npm conventions :)   Never mind. Leave it as it is